packages/bitcore-lib/test/publickey.js

```diff
@@ -451,4 +451,36 @@ describe('PublicKey', function() {
 
   });
 
+  describe('#isValidTaproot', function() {
```

Comment:
Consider including Chai assertions that duplicate testing behavior (assert on length, assert on characters, etc)